src/cgame/cg_rocket.cpp

```diff
@@ -77,23 +76,10 @@ void CG_Rocket_Init( glconfig_t gl )
 	Rocket_RegisterProperty( "locked-marker-color", "red", false, false, "color" );
 
 	// Preload all the menu files...
-	len = trap_FS_FOpenFile( rocket_menuFile.Get().c_str(), &f, fsMode_t::FS_READ );
+	std::error_code err;
```

Comment:
Is this line really useful? (and if it is, why isn't it here in the next function?)

Reply:
Removed the line.